release_dashboard/test/widgets/release_completed_test.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/proto.dart' as pb;
+import 'package:conductor_ui/state/status_state.dart';
+import 'package:conductor_ui/widgets/release_completed.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+
+import '../fakes/services/fake_conductor.dart';
+import '../src/test_state_generator.dart';
+
+void main() {
+  late pb.ConductorState stateWithoutConflicts;
+
+  setUp(() {
+    stateWithoutConflicts = generateConductorState();
+  });
+  testWidgets('Render all elements correctly', (WidgetTester tester) async {
+    await tester.pumpWidget(ChangeNotifierProvider(
+      create: (context) => StatusState(conductor: FakeConductor(testState: stateWithoutConflicts)),
+      child: MaterialApp(
+        home: Material(
+          child: Column(
+            children: <Widget>[
+              Builder(builder: (context) {
```

Comment:
does this need to be a `Builder` here?

Reply:
Good point, don't know why I added it here. Removed!